recipes/tests.rb

```diff
@@ -242,51 +242,53 @@
 ###################
 # EFA - Intel MPI
 ###################
-case node['cfncluster']['os']
-when 'alinux', 'centos7', 'alinux2'
-  execute 'check efa rpm installed' do
-    command "rpm -qa | grep libfabric && rpm -qa | grep efa-"
-    user node['cfncluster']['cfn_cluster_user']
-  end
-  execute 'check intel mpi installed' do
-    command "rpm -qa | grep intel-mpi"
-    user node['cfncluster']['cfn_cluster_user']
-  end
-when 'ubuntu1604', 'ubuntu1804'
-  case node['cfncluster']['cfn_node_type']
-  when 'MasterServer'
-    execute 'check ptrace protection enabled' do
-      command "sysctl kernel.yama.ptrace_scope | grep 'kernel.yama.ptrace_scope = 1'"
+if node['conditions']['efa_supported'] && node['conditions']['intel_mpi_supported']
```

Comment:
I can be wrong but I don't think that `node['conditions']` is not available here, please correct me

Reply:
This is counter to what I've seen. Why would it not be available?